config/routes.rb

```diff
@@ -34,6 +34,8 @@
     get '/country/:iso/compare(/:iso_to_compare)', to: 'country#compare', as: 'compare_countries'
     get '/country/:iso/protected_areas', to: 'country#protected_areas', as: 'country_protected_areas'
 
+    get '/global_statistics', to: 'global_statistics#show'
```

Comment:
change this accordingly to the controller action name comment

Reply:
changed